Externals.cfg

```diff
@@ -21,7 +21,7 @@ protocol = git
 repo_url = https://github.com/NOAA-GSL/ufs-weather-model
 # Specify either a branch name or a hash but not both.
 #branch = gsd/develop
-hash = f204bfd
+hash = 74f8c25
```

Comment:
Is this hash from the RRFS_dev branch? Maybe change the branch comment above to reduce ambiguity.